control/checkbox.go

```diff
@@ -0,0 +1,45 @@
+package control
+
+import (
+	"github.com/jbowtie/gokogiri/xml"
+	"strings"
+)
+
+const checkBoxAttributeKey = "val"
+
+func newCheckBox(checkMark xml.Node, textNodes *[]xml.Node) *checkBox {
+	return &checkBox{checkMark: checkMark, textNodes: textNodes}
+}
+
+type checkBox struct {
+	checkMark xml.Node
+	textNodes *[]xml.Node
+}
+
+func (c *checkBox) isChecked() bool {
+	if c.checkMark.Attr(checkBoxAttributeKey) == "1" {
+		return true
+	}
+	return false
```

Comment:
Super minor, but thoughts about simplifying this to `return c.checkMark.Attr(checkBoxAttributeKey) == "1"`?